packages/ui-kit/src/components/Autocomplete/Autocomplete.css

```diff
@@ -0,0 +1,3 @@
+li.Mui-focusVisible {
```

Comment:
We can make it work in the sass world by using the following selector:
```scss
// packages/ui-kit/src/components/Autocomplete/Autocomplete.module.scss

.autocompleteList {
  li {
    &[class^='Mui-focusVisible'] {
      background-color: var(--propel-colors-blue-25);
    }
  }
}
```

It's important to use `contains` instead of `equal` cause otherwise it will fail to select the item that has multiple class names, e.g. `Mui-focusVisible Mui-focused`